about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Who Am I?</title>
+    <link rel="stylesheet" href="styles/style.css">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="Portfolio.html">Portfolio</a></li>
+          <li><a href="Code-Journal.html">Code Jounal</a></li>
+          <li><a href="About.html">About </a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <article class="text-align-center">
+
+      <h1>Discover who I am aside from a Student Full-Stack Developer</h1>
+      <h2>My passions and personality</h2>
+   </article>
+
+   <article class="grey-background text-align-center">
+     <h1>A few fun facts about myself</h1>
+     <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English.</p>
+
+    <img src="" alt="">
+
+  </article>
+
+  <article class="">
```

Comment:
We do not need the `class` attribute on this element as it has not been set to any value.